src/components/TodoItem.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "./TodoItem.css";
+class TodoItem extends React.Component {
+  onItemChecked = e => {
+    this.props.onItemChecked(this.props.item.id);
+  };
+  onButtonClick = e => {
+    this.props.onItemDelete(this.props.item.id);
+  };
```

Comment:
I would name props `onChecked` and `onDelete` (omitting `Item`). Rename `onButtonClick` function as well